src/applications/disability-benefits/new-686/config/generic-schema.js

```diff
@@ -27,4 +27,31 @@ export const genericSchemas = {
     enum: states50AndDC.map(state => state.value),
     default: states50AndDC.map(state => state.label),
   },
+  fullName: {
+    type: 'object',
+    properties: {
+      first: {
+        type: 'string',
+        minLength: 1,
+        maxLength: 30,
+        pattern: '^(?!\\s)(?!.*?\\s{2,})[^<>%$#@!^&*0-9]+$',
```

Comment:
Might be worth pulling this pattern into a variable and using it across all three names. Also, it might be worth double-checking that the error message is helpful when there's a validation error because of this new pattern.